job-descriptions/software-engineer.md

```diff
@@ -1,46 +1,75 @@
 
-### About the Role
+## About the role
 
-You will help build Sourcegraph, a multi-tier application (web, CLI, API, data stores, services) written primarily in TypeScript (frontend) and Go (backend). You will have the freedom to creatively tackle various CS challenges while building products that are fundamental to the growth and success of the business.
+You will
 
-You will have the opportunity to interact directly with our customers to help them deploy/configure our software, and solve issues that they encounter.
+- Help build Sourcegraph, a multi-tier application (web, CLI, browser extensions, API, data stores, services) written primarily in TypeScript (frontend) and Go (backend).
+- Have the freedom to creatively tackle various CS challenges while building products and infrastructure that are fundamental to the growth and success of the business.
+- Have the opportunity to interact directly with our customers to help them deploy/configure our software, and solve issues that they encounter.
+- Plan your own work each month based on company goals and customer feedback.
+- Publish blog posts and give conference talks about your work at Sourcegraph.
 
-As a senior member of the team, you will help set the technical direction of various projects as well as mentor other team members. This is a full-stack role and ideal candidates should feel comfortable (though not equally skilled) at all layers of the stack.
+As a senior member of the team, you will
 
-**This position can be onsite or remote**
+- Help set the technical direction of various projects.
+- Mentor and teach other team members.
 
-### Example projects:
+This is a full-stack role and ideal candidates should feel comfortable (though not equally skilled) contributing to any part of our codebase.
 
-- Reduce P95 latency of "code intelligence" requests (e.g. hover tooltips, find references, go-to-definition)
-- Scale our language analysis cluster to meet traffic demands; mitigate infrastructure costs
-- Design and implement a system to perform fast, real-time, cross-repository `git grep`
+### Ideal candidates have
 
-### Responsibilities
+- A track record of delivering high-quality products with attention to scalability and UX.
+- Strong web/JavaScript/TypeScript/Go fundamentals.
+- Experience working with APIs and distributed systems.
+- Passion for the craft of software development and good engineering practices.
 
-- Design and build multi-tier web & desktop applications using Go & TypeScript
-- Combine system metrics, user feedback, and intuition to plan improvements
-- Create simple and performant APIs and services based on multiple underlying data sources
-- Lead and mentor other team members while driving technical roadmaps and decisions for various projects
+### Example projects

```

Comment:
@felixfbecker @KattMingMing @ijsnow @keegancsmith @slimsag @ggilmore 

Let me know if you can think of examples that are better than the ones I already have

Reply:
These look great - There could also be open tickets on the browser extension repository that would be good for somebody interviewing to tackle as well. Once CXP is in production we should update the example to include creating an extension.

Reply:
Actually Nick just added those to the projects list yesterday: https://docs.google.com/document/d/1kIxMMFTNlCQ2ofl2aq9K3GFFeGkl5JgSdpR-6mor_Ig/edit#! This is just the job description to give examples to people